scripts/superdesk-authoring/authoring.js

```diff
@@ -1065,6 +1071,19 @@
                         });
                 }
 
+                function _previewHighlight(_id) {
+                    api.generate_highlights.save({}, {'package': _id, 'preview': true})
+                    .then(function(response) {
+                        $scope.highlight_preview = $sce.trustAsHtml(response.body_html);
```

Comment:
@vied12 we don't have special way of dealing with trusted html? I remember that i saw you asking about that somewhere?